docs/user.md

```diff
@@ -0,0 +1,183 @@
+# How to create a new db cluster
```

Comment:
Can we write "Postgres" cluster? Especially with the wording database in Postgres being a logical unit within a Cluster this can be confusing.